### ISSUE TITLE
super focus doesn't use the whole screen

### ISSUE THREAD

**Issue by anatoliibakhtin** [ryanpcmcquen/obsidian-focus-mode#34]
Hello dear developer!

If you open several notes located not only horizontally, but also vertically, the super focus mode does not stretch the active note completely:
![не раскрывается во весь экран](https://user-images.githubusercontent.com/71871141/187041437-abe95a7c-0fc4-4317-8bc3-7d71b700fb67.png)

it would be great if all the active note will also occupy the whole screen. 
Thank you for your attention 🌼

**Comment by ryanpcmcquen** [comment]
Would accept a PR implementing this.

**Comment by ryanpcmcquen** [comment]
Is this still present on the latest version?

**Comment by ronrdev** [comment]
I'm a big user of super focus
There is a focus on active pane hotkey feature in the pane relief plugin that works. It doesn't close sidebars or hide icons the way this plugin does, but whatever it is doing might solve this issue, as it successfully hides all over tabs/panes without blank space or artifacts. 
https://github.com/pjeby/pane-relief

**Comment by ryanpcmcquen** [comment]
Will you try the latest version and let me know if you have any issues?
https://github.com/ryanpcmcquen/obsidian-focus-mode/releases/tag/1.11.0

**Comment by anatoliibakhtin** [comment]
@ryanpcmcquen Better already !!! it's like the groups are wrap like slides, 1, 2... 10 groups work fine if there are no "pillars" between them, just like in Blender you cannot collapse windows, but it goes into focus mode just fine.

https://user-images.githubusercontent.com/71871141/199050192-646250f0-8618-425c-97c2-3739ca47b557.mp4

With the advent of tabs the focus mode became less urgent, what I have now is quite satisfactory, thank you for your work ! If you want to close this thread, that's fine with me.

p.s. @ronrdev !!!😻

**Comment by ryanpcmcquen** [comment]
Have you tried Super Focus Mode?

**Comment by ronrdev** [comment]
Hello. I have the same experience @anatoliibakhtin  posted. 

Super focus seems to have trouble with certain panel configurations. I think it's because the plugin has the two different options.

focus mode hides interface clutter (distraction free)
super focus add a maximize active pane 

I believe super focus mode might have to work different than the regular focus mode. In the regular focus mode you are hiding the note header, but not the tab bar so all the panels work well.

It seems like in super focus mode you should hide the tab bar because there will only be one note pane visible anyway. Without needing to show any tab bars it should work better.

Attached video shows super focus mode, and then activating maximize active panel command from pane relief with a hotkey. The tab bars hide, the active pane header stays.

https://user-images.githubusercontent.com/13022919/199088409-ab2d2d18-e38b-4040-bcac-824f7337d057.mp4

**Comment by anatoliibakhtin** [comment]
@ronrdev Thank you for your recommendation, the plugin Pane Relief !! You suggested is what I was looking for, I've been waiting for a very long time !!!😻

**Comment by ryanpcmcquen** [comment]
> Hello. I have the same experience @anatoliibakhtin posted.
> 
> Super focus seems to have trouble with certain panel configurations. I think it's because the plugin has the two different options.
> 
> focus mode hides interface clutter (distraction free) super focus add a maximize active pane
> 
> I believe super focus mode might have to work different than the regular focus mode. In the regular focus mode you are hiding the note header, but not the tab bar so all the panels work well.
> 
> It seems like in super focus mode you should hide the tab bar because there will only be one note pane visible anyway. Without needing to show any tab bars it should work better.
> 
> Attached video shows super focus mode, and then activating maximize active panel command from pane relief with a hotkey. The tab bars hide, the active pane header stays.
> 
>  test-2022-10-31_14.52.47.mp4 
> 
> I also noticed a bug to exit super focus mode you click the icon, if shift clicking to exit it does not reveal toolbar icons. Attached video to show icons remain hidden if using shift click to exit. Sorry to post two issues in one thread. I can create a separate issue later if prefer, or this isn't a big bug because a regular click works.
> 
>  icons-2022-10-31_15.04.23.mp4

The maximize part is a bug, will look into it.

As far as Shift + Click, that is expected behavior, it only toggles 'Super Focus Mode', meaning if you were already in 'Focus Mode' you will remain in it. This way you can move from any of the three statuses to any other status with one interaction.

**Comment by ryanpcmcquen** [comment]
@anatoliibakhtin will you try the latest release?
https://github.com/ryanpcmcquen/obsidian-focus-mode/releases/tag/1.11.1

**Comment by ryanpcmcquen** [comment]
@anatoliibakhtin @ronrdev
This should be better: https://github.com/ryanpcmcquen/obsidian-focus-mode/releases/tag/1.11.2

**Comment by ryanpcmcquen** [comment]
@anatoliibakhtin @ronrdev
This should be _even_ better: https://github.com/ryanpcmcquen/obsidian-focus-mode/releases/tag/1.11.3

**Comment by ronrdev** [comment]
Thanks for working on this!

I just tested it with many panel configs and works well. Now click vs shift click toggles different states of focus and super focus mode so was confusing at first, but works well now I understand the use.

_I don't use stacked tabs/sliding panes but I tested it and the vertical tabs are still visible in super focus mode and clickable to pull focus on other notes. Everything else is hidden._ 

Unless I'm missing a click pattern to do this, the only inconsistent thing I noticed is that focus mode closes and opens sidebars. When entering super focus mode with sidebars open, the plugin does not reopen them when toggling back. 

Attached video shows focus mode toggling sidebars, super focus closes them but they never open back up unless manually opened with button or separate hotkey.
https://user-images.githubusercontent.com/13022919/199372681-5386a16d-4dcf-4348-9f5b-a8d324de8252.mp4

**Comment by ryanpcmcquen** [comment]
Would you mind opening a new bug for this? The plugin stores the state of sidebars ... perhaps there was a regression introduced there with Obsidian 1.x. Are you referring to the default left and sidebar only?

**Comment by ronrdev** [comment]
Sure. I opened a new issue for super focus mode loosing state of open sidebars.
